i18n/en.json

```diff
@@ -802,21 +802,13 @@
     "shortName": "Mode",
     "longName": "Operational Mode of Autopilot"
   },
-  "steering.autopilot.targetHeadingNorth": {
-    "shortName": "Target Heading North",
-    "longName": "Target Heading North in Degrees True"
+  "steering.autopilot.targetAngle": {
+    "shortName": "Target (relative) heading",
+    "longName": "TTarget heading for autopilot, relative to magnetic North or Apparent wind +port -starboard"
```

Comment:
Ideally, `shortName` should be really short. Like "HTS" for Heading to Steer. `longName` could then be "Target Heading (relative)". What's in `longName` currently better fits in the `description` field below, IMO.

But honestly this is probably a bigger discussion than this PR. The i18n stuff still needs work. Perhaps in addition to `shortName` and `longName` we need `abbreviatedName` as well?

If nothing else, there's an extra 'T' in `longName` that should be removed :smiley: